CMakeLists.txt

```diff
@@ -64,15 +64,40 @@ message(STATUS "COMMON_OS_LIBS       = ${COMMON_OS_LIBS}")
 message(STATUS "PYTHON2_BUILD        = ${PYTHON2_BUILD}")
 message(STATUS "PYTHON3_BUILD        = ${PYTHON3_BUILD}")
 
+# Now build nupic_netapi project.
+set(HTM_LIB_INCLUDES     ${REPOSITORY_DIR}/htm/src)    # for htm/src/nupic/xxx/*.hpp 
+set(NETAPI_LIB_INCLUDES  ${REPOSITORY_DIR}/netapi/src)  # for netapi/nupic/xxx/*.hpp
```

Comment:
>what name should be use? 
- NetworkAPI of course is the real name but as I was doing this movement, I quickly discovered this is too long to type.   NetworkAPI is not a great name...because it does imply something to do with the internet. But I think we are stuck with that.
- netapi is what I shortened it to.  But I agree not a great name either.
- I had never heard the term "napi" other than when you used it.  It took me a while to figure out what it meant.  But it is nice and short. If it is being used elsewhere then that probably should be the name.